.gitignore

```
@@ -8,3 +8,5 @@ Gemfile.lock
 /travis/*.lock
 /test/database_local.yml
 .idea
+debug.log
+test/db/*
```

```suggestion
/test/db/*
```

This makes it explicit that the path only comes from project root :)